### ISSUE TITLE
Failed to build application for the Web.

### ISSUE THREAD

**Issue by Lb0728** [syncfusion/flutter-examples#66]
...
LBdeMacBook-Pro:flutter_athena lb$ flutter run -d chrome
 
Launching lib/main.dart on Chrome in debug mode...
                                                                        
AssetNotFoundException: syncfusion_flutter_charts|lib/charts.ddc.js     
Error compiling dartdevc module:syncfusion_flutter_charts|lib/charts.ddc.js

packages/syncfusion_flutter_charts/src/circular_chart/utils/helper.dart:22:9: Error: Expected 0 type arguments.
  final Point<double> innerRadiusStartPoint = Point<double>(
        ^^^^^
packages/syncfusion_flutter_charts/src/circular_chart/utils/helper.dart:22:47: Error: 'Point' is imported from both 'dart:math' and 'dart:ui'.
  final Point<double> innerRadiusStartPoint = Point<double>(
                                              ^^^^^
packages/syncfusion_flutter_charts/charts.dart:7:1: Error: 'Point' is imported from both 'dart:math' and 'dart:ui'.
import 'dart:ui';
^^^^^
packages/syncfusion_flutter_charts/src/circular_chart/utils/helper.dart:25:9: Error: Expected 0 type arguments.
  final Point<double> innerRadiusEndPoint = Point<double>(
        ^^^^^
packages/syncfusion_flutter_charts/src/circular_chart/utils/helper.dart:25:45: Error: 'Point' is imported from both 'dart:math' and 'dart:ui'.
  final Point<double> innerRadiusEndPoint = Point<double>(
                                            ^^^^^
packages/syncfusion_flutter_charts/src/circular_chart/utils/helper.dart:29:9: Error: Expected 0 type arguments.
  final Point<double> radiusStartPoint = Point<double>(
        ^^^^^
packages/syncfusion_flutter_charts/src/circular_chart/utils/helper.dart:29:42: Error: 'Point' is imported from both 'dart:math' and 'dart:ui'.
  final Point<double> radiusStartPoint = Point<double>(
                                         ^^^^^

                                                                        
Failed after 234ms                                                      
Building application for the web...                                10.1s
Failed to build application for the Web.

...

**Comment by dharanidharandharmasivam** [comment]
Hi @Lb0728 ,

Thanks for your interest in our Syncfusion Flutter widgets. Since Flutter web is in beta, we haven’t provided web support yet for our widgets, so the compilation issue has occurred. However, we are working for the web also in parallel, once Flutter web has moved to the stable channel, we will also roll out our Flutter web support. We appreciate your patience until then. 

Thanks,
Dharani.

**Comment by jans-y** [comment]
Hi @dharanidharandharmasivam,

Thank you for your response.

We have bought the license thinking that this will work since this is a pure Dart library without native dependencies. It's very critical for us to have it as soon as possible. Your calendar plugin fortunately worked on the web for us.

I believe it will take at least until May 2020 (Google I/O 2020) for web to be on the stable channel.

We have a project where Charts are one of the main features, and **we can't proceed** to the next stage without Web support. We are only using line/area charts which is what most of the users probably also need.

It would be a great help for us if we could at least compile the app and have at least passive line charts without interaction.

Thank you.

**Comment by dharanidharandharmasivam** [comment]
Hi @Lb0728 ,

Thanks for the revert. Based on your requirement we have fixed the compilation issues in chart widget for flutter web, and now the requested series will get rendered. Kindly note that only the basic things we have covered with the issue fix, yet we need to test all user interactions and other cases for all the series. Kindly find the package and sample from the below link.

Sample: https://www.syncfusion.com/downloads/support/directtrac/general/ze/line_series_web1917727354 
Package: https://www.syncfusion.com/downloads/support/directtrac/general/ze/intermediate-1956211940 

We will include the fix in our volume 1 main release which is expected to be out at the end of March 2020. Kindly use the above package to resolve the reported issue and replace the local path for our flutter chart widget dependency in pubspec.yaml file.

Thanks,
Dharani.
